src/java.base/share/classes/java/nio/X-Buffer.java.template

```diff
@@ -2247,7 +2247,7 @@ public abstract sealed class $Type$Buffer
             throw new IllegalArgumentException("Index less than zero: " + index);
         if (unitSize < 1 || (unitSize & (unitSize - 1)) != 0)
             throw new IllegalArgumentException("Unit size not a power of two: " + unitSize);
-        if (unitSize > 8 && !isDirect())
+        if (unitSize > 1 && !isDirect())
```

Comment:
This will require an update to the implNote as it documents 8.

Reply:
Thanks, good catch